support-frontend/assets/components/subscriptionCheckouts/subscriptionSubmitButton.scss

```diff
@@ -2,5 +2,16 @@
 
 .component-submit-button--margin {
     max-width: gu-span(5);
-    margin: 0 $gu-h-spacing / 2 $gu-v-spacing * 2;
+    margin-top: 0; 
+    margin-right: $gu-h-spacing / 2; 
+    margin-bottom: $gu-v-spacing * 2;
+    margin-left: $gu-h-spacing / 2;
+}
+
+.show {
+    display: block;
+}
+
+.hide {
+    display: none;
```

Comment:
ah in fact these are pretty much what i meant about utility classes, is it best practice to reuse these?